packages/builder/src/helpers/bindings.ts

```diff
@@ -0,0 +1,74 @@
+import { makePropSafe } from "@budibase/string-templates"
```

Comment:
This has been moved from `packages/builder/src/components/design/settings/controls/DataSourceSelect/DataSourceSelect.svelte`